joomla/joomla-cms
# Calendar change event

Pull Request for Issue #32694 and #17515 for joomla 3

### Summary of Changes

make calendar.js trigger "change" event when user select new date


### Testing Instructions

Add to Isis index.php:
```html
<script type="module">
if (window.jform_publish_up){
  window.jform_publish_up.addEventListener('change', () => {
    console.log('change');
  })
}
</script>
```

Go to Article editing, and try to change the publish_up date.
In browser console you should get "change" message on each change


### Actual result BEFORE applying this Pull Request
No message


### Expected result AFTER applying this Pull Request
Message


### Documentation Changes Required
none
